<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IRC - ledins_a</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="userInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="messages">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>761</width>
      <height>561</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>630</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QListView" name="usersList">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>80</y>
      <width>201</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Users</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chat</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>761</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>irc</string>
    </property>
    <addaction name="actionConnection_to_server"/>
    <addaction name="actionJoin_chanel"/>
   </widget>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnection_to_server">
   <property name="text">
    <string>connect to server</string>
   </property>
  </action>
  <action name="actionJoin_chanel">
   <property name="text">
    <string>join chanel</string>
   </property>
  </action>
  <action name="actionCoucou">
   <property name="text">
    <string>coucou</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
